Framework/Core/src/WorkflowHelpers.cxx

```diff
@@ -263,6 +264,8 @@ void WorkflowHelpers::injectServiceDevices(WorkflowSpec& workflow, ConfigContext
     {ConfigParamSpec{"aod-file", VariantType::String, {"Input AOD file"}},
      ConfigParamSpec{"aod-reader-json", VariantType::String, {"json configuration file"}},
      ConfigParamSpec{"time-limit", VariantType::Int64, 0ll, {"Maximum run time limit in seconds"}},
+     ConfigParamSpec{"orbit-offset-enumeration", VariantType::Int64, 0ll, {"initial value for the orbit"}},
+     ConfigParamSpec{"orbit-multiplier-enumeration", VariantType::Int64, 1ll, {"multiplier to get the orbit from the counter"}},
```

Comment:
should not default be 0 also here, i.e. 1st orbit will stay 0?

Reply:
yes, I changed it to 0 as discussed to indicate that the value was not provided.